README.md

```diff
@@ -23,11 +23,12 @@ setenv CMSSW_GIT_REFERENCE /cvmfs/cms.cern.ch/cmssw.git.daily
 git cms-init
 
 ```
- * Top branch needed to run updated double b tagger training on AK8 jets
+ * Top branch needed to run updated double b tagger training on AK8 jets, and latest Run II MET filters
 ```
 git fetch --tags btv-cmssw
 
 git cms-merge-topic -u cms-btv-pog:BoostedDoubleSVTaggerV3-WithWeightFiles-v1_from-CMSSW_8_0_8_patch1
```

Comment:
I don't think these producers are added to CMSSW yet. I didn't change the CMSSW version in the recipe, I kept 8_0_16. But I used the official JetMET pog recipe, so I think this is the correct way to do.